packages/nodes-from-anchor/src/index.ts

```diff
@@ -1,12 +1,18 @@
 import { RootNode } from '@kinobi-so/nodes';
 
 import { IdlV00, rootNodeFromAnchorV00 } from './v00';
+import { IdlV01, rootNodeFromAnchorV01 } from './v01';
 
 export * from './discriminators';
 export * from './v00';
+export * from './v01';
 
-export type AnchorIdl = IdlV00;
+export type AnchorIdl = IdlV00 | IdlV01;
 
 export function rootNodeFromAnchor(idl: AnchorIdl): RootNode {
-    return rootNodeFromAnchorV00(idl);
+    if ((idl as IdlV01).address) {
```

Comment:
I think it might be more robust to rely on `idl.metadata.spec` since this is the new field they've introduced to keep track of the version of the spec.

So if `idl?.metadata?.spec` is undefined, we fallback to `v00`, otherwise we can parse that version and create a switch/case for all supported versions. Right now that would only be `"0.1.0"` and we can throw an error for any other version.